eng/testing/workloads-browser.targets

```diff
@@ -63,22 +61,20 @@
            Text="Expected to find either one or two in $(LibrariesShippingPackagesDir): @(_RuntimePackNugetAvailable->'%(FileName)%(Extension)')" />
 
     <ItemGroup>
-      <_BuildVariants Include="multithread" Condition="'$(_DefaultBuildVariant)' != '.multithread.'" />
-
-      <_NuGetsToBuild Include="$(LibrariesShippingPackagesDir)Microsoft.NETCore.App.Runtime.Mono.%(_BuildVariants.Identity).$(RIDForWorkload).$(PackageVersionForWorkloadManifests).nupkg"
+      <_NuGetsToBuild Include="$(LibrariesShippingPackagesDir)Microsoft.NETCore.App.Runtime.Mono.$(_BuildVariant)$(RIDForWorkload).$(PackageVersionForWorkloadManifests).nupkg"
```

Comment:
If I'm reading it correctly, we can reduce the code to have just one `_NuGetsToBuild` element, am I correct? 

My understanding
- Just one of `_NuGetsToBuild` is included in all cases
- They differ only in 
  - Descriptor - could be composed dynamicly
  - Identity - could always incude `_BuildVariant` 
  - Properties - passing empty/false `WasmEnableThreads` do no harm